door43_tools/templaters.py

```diff
@@ -33,6 +38,9 @@ def run(self):
         self.rc = RC(self.source_dir)
         with open(self.template_file) as template_file:
             self.template_html = template_file.read()
+            soup = BeautifulSoup(self.template_html, 'html.parser')
+            soup.body['class'] = self.resource_type
```

Comment:
A couple observations here. 
* First, there is a bug that has been around a long time that results in 2 sets of `body` tags in the final output. Does this update the correct one?
* Second, I think you might be better off adding  another class to the body tag rather than replacing all existing classes.